reactjs/src/services/role/dto/getAllPermissionsOutput.ts

```diff
@@ -0,0 +1,12 @@
+
+
+    export interface GetAllPermissionsOutput {
+        name: string;
+        displayName: string;
+        description: string;
+        id: number;
+    }
+
+   
+
+
```

Comment:
There are unnecessary empty lines and whitespaces in files. I don't write review to all files, but you can check all of them, and maybe you can prefer to use a linter.

Reply:
Resolved